mode/type/standalone/repository/provider/jdbc/src/main/java/org/apache/shardingsphere/mode/repository/standalone/jdbc/JDBCRepository.java

```diff
@@ -122,7 +122,17 @@ public List<String> getChildrenKeys(final String key) {
     
     @Override
     public boolean isExisted(final String key) {
-        return !Strings.isNullOrEmpty(query(key));
+        try (
+                Connection connection = dataSource.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(repositorySQL.getSelectByKeySQL())) {
+            preparedStatement.setString(1, key);
+            try (ResultSet resultSet = preparedStatement.executeQuery()) {
+                return resultSet.next();
```

Comment:
Perhaps we could replace next() with hasNext()?

Reply:
There is no hasNext() method